variables.tf

```diff
@@ -121,6 +121,17 @@ variable "runners_volumes" {
   default     = ["/cache"]
 }
 
+variable "runners_volumes_tmpfs" {
+  type = "list"
+  default = []
+}
+
+
+variable "runners_services_volumes_tmpfs" {
+  type = "list"
```

Comment:
Please add a description, the `auto_docs` script will update the description in the readme.

Reply:
Added description and run autodocs script